wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -22,25 +22,18 @@ import Contacts
 import WireDataModel
 import WireCommonComponents
 
-final class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
```

Comment:
If you would like to remove DynamicTypeCapable from this class please make sure all of the labels here use the new initializer from DynamicFontLabel. 

`let nameLabel = DynamicFontLabel(fontSpec: .normalSemiboldFont, color: LabelColors.textDefault)` still uses the old initializer

Reply:
I agree but this specific label uses the deprecated initializer. If you would like to remove conformance to DynamicTypeCapable you need to make sure you're using the new initializers on all labels. The nameLabel still uses deprecated initializer (fontSpec)

The new one would look like this:

`let nameLabel = DynamicFontLabel(style: .body , color: LabelColors.textDefault)` // .body is just an example, please make sure we're using the correct font style.

Reply:
got it. Done